PEM-8001: update of okta-saml pages (#7273)
spectrocloud/librarium: docs/docs-content/user-management/saml-sso/palette-sso-with-okta.md

   member of that start with `palette-` in the `groups` claim of the ticket. For Palette SSO, Palette makes the user a
   member of a Palette team that has an identical name.

   | Claim Name | Include in token type | Value Type | Filter                  | Disable claim | Include In |
   | ---------- | --------------------- | ---------- | ----------------------- | ------------- | ---------- |
   | `groups`   | ID Token (Always)     | Groups     | Starts with: `palette-` | Unchecked     | Any scope  |

   ![Claims Result](/oidc-okta-images/oidc-okta_claims-result.webp)

7. At the top of the page, select the **Back to Authorization Servers** breadcrumb to return to your list of servers.

8. The authorization server you created is displayed in the list. Select the **Issuer URI** shown and copy it to the
   clipboard. Save this value, as you will need it later.

   ![Get Issuer URI](/oidc-okta-images/oidc-okta_get-issuer-uri.webp)

9. Select the Palette OIDC application and navigate to the **Access Policies** tab. Select **Add Policy**.

   ![Add Access Policy](/oidc-okta-images/oidc-okta_add-access-policy.webp)

> 10. Set the **Name** and **Description** fields to `Palette`, then change the **Assign to** option to **The following
>     clients**. In the drop-down, search for the Okta application you created in step 3; in our example, we used
>     `Spectro Cloud Palette OIDC`. Once selected, click **Create Policy**.

Reviewer: ```suggestion
10. Set the **Name** and **Description** fields to `Palette`, then change the **Assign to** option to **The following
    clients**. In the drop-down, search for the Okta application you created in step 3 of [Create the Okta Application](#create-the-okta-application); in our example, we used
    `Spectro Cloud Palette OIDC`. Once selected, click **Create Policy**.
```

   | Issuer URL    | The Issuer URI that you saved in step 8 of [Create an Okta Authorization Server](#create-an-okta-authorization-server).                                                                                                        |
   | Client ID     | The client identifier that you saved in step 12 of [Create the Okta Application](#create-the-okta-application).                                                                                                                |
   | Client Secret | The shared secret that you generated in step 13 of [Create the Okta Application](#create-the-okta-application)..                                                                                                               |
   | Default Teams | Leave blank if you do not want users without group claims to be assigned to a default group. If you do, enter the desired default group name. If you use this option, be careful with how much access you assign to the group. |

>    | Scopes        | Keep `openid`, `profile` and `email` as the default.                                                                                                                                                                           |

Reviewer: ```suggestion
   | Scopes        | Keep `openid`, `profile`, and `email` as the default.                                                                                                                                                                           |
```

   | Email         | Keep `email` as the default.                                                                                                                                                                                                   |
   | First Name    | Set this to `u_first_name`.                                                                                                                                                                                                    |
   | Last Name     | Set this to `u_last_name`.                                                                                                                                                                                                     |
   | Spectro Team  | Keep `groups` as the default.                                                                                                                                                                                                  |

![Enable Palette OIDC SSO Part 1](/oidc-okta-images/oidc-okta_configure-palette-oidc-part1.webp)

> ![Enable Palette OIDC SSO Part 2](/oidc-okta-images/oidc-okta_configure-palette-oidc-part2.webp)

Reviewer: Additional indentation needed. Images not aligned.

2. From the left main menu, select **Users & Teams**.

3. Navigate to the **Teams** tab and select **Create Team**.

   ![Create Palette Team](/oidc-okta-images/oidc-okta_create-team.webp)

4. Enter `palette-tenant-admins` for the **Team name**. You do not need to add any members, as members will be added
   automatically with SSO. Select **Confirm** to create the team.

   ![Name Palette Team](/oidc-okta-images/oidc-okta_name-team.webp)

5. A message states that the team was created. Select the newly created `palette-tenant-admins` team to review the
   team's details.
6. To give the team administrative access to the entire tenant and all projects within the tenant, select the **Tenant
   Roles** tab and click **Add Tenant Role**.

   ![Palette Tenant Roles](/oidc-okta-images/oidc-okta_tenant-roles.webp)

7. In the **Add Roles** dialog, select **Tenant Admin**. Click **Confirm** to add the role.

   ![Add Tenant Role](/oidc-okta-images/oidc-okta_add-tenant-role.webp)

A message states that **Roles have been updated**. Repeat this procedure for any other necessary teams, taking care to
ensure they are given the appropriate permissions.

> ## Validate

Reviewer: Can't comment on the first step since it's unchanged, but same changes as previous article.

## Validate

To verify that SSO is working as expected, you must log in to Palette as an Okta user who is a member of the [Okta application](#create-the-okta-application) you recently created.

1. Log out of Palette by selecting **Logout** in the **User Menu** at the top right.
   ![User Logout](/oidc-okta-images/oidc-okta_user-logout.webp)

2. The Palette login screen now displays a **Sign in** button and no longer presents a username and password field. Select **Sign in** to log in via SSO.

:::tip

 Below the **Sign In** button, there is an **SSO issues? Use your password** link. Use this link if you need to bypass SSO
   and log in with a Palette account. 

:::

![User SSO Login](/oidc-okta-images/oidc-okta_palette-login.webp)

... And then the rest would be the same :) HOWEVER, step 3 confuses me a tad:

`Make sure you log in as a user that is a member of the `palette-tenant-admins` group in Okta`

I don't think we ever made a `palette-tenant-admins` group in Okta? Or, at least, we did not call it that. Do they just need to be a member of the Okta app we created?

   :::

> 4. You are now automatically added to the `palette-tenant-admins` team in Palette. To verify, navigate to the left main

Reviewer: ```suggestion
4. Upon logging in, you are automatically added to the `palette-tenant-admins` team in Palette. To verify, navigate to the left main
```
